search/index.html.in

```diff
@@ -1,55 +1,59 @@
 <!doctype html>
-<html lang="en">
+<html lang="en" id="search">
 <head>
   <meta charset="utf-8">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <title>Chaise :: Search</title>
   <script src="/gtm-id.js"></script>
   %ASSETS%
 </head>
-<body ng-app="ermrestApp" style="overflow:hidden;">
-    <!-- Overlay for fixed sidebar -->
-    <div class="sidebar-overlay"></div>
-    <!-- Row 1: Logo and Main navigation -->
-    <navbar></navbar>
-    <div id="main-wrapper" style="height:100vh; overflow-y:auto;">
-        <div id="main-content" class="col-xs-6 col-sm-6 col-md-7 col-lg-8 col-xl-9" style="margin-left:40px;">
-            %TEMPLATES%
+<body style="overflow:hidden;">
+    <div ng-app="configure-search"></div>
```

Comment:
I'm assuming this line and also removing the `ng-app` from `body` are the only changes in this file, right? You've fixed the indentation and it's showing that you've changed the whole file.

Reply:
Right. Removing `ng-app` from body and adding this new div for the configuration app. As well as wrapping the content of the app in another div with the `app-container` class.